client/commonFramework/execute-challenge-stream.js

```diff
@@ -18,7 +18,7 @@ window.common = (function(global) {
     const originalCode = code;
     const head = common.arrayToNewLineString(common.head);
     const tail = common.arrayToNewLineString(common.tail);
-    const combinedCode = head + code + tail;
+    const combinedCode = head + code + '\n' + tail;
```

Comment:
I am yet to test this, but I am wondering if

`const combinedCode = head + '\n' + code + '\n' + tail;`

would be a full proof to address any code that may have comment on the top.
Not that this is necessary but wouldn't this be good?